index.html

```diff
@@ -0,0 +1,268 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://necolas.github.io/normalize.css/8.0.1/normalize.css">
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js" defer></script>
+  <title>Abstract</title>
+</head>
+<body>
+  <div class="container">
+    <!-- ######################################### SCROLL TO TOP ######################################## -->
+    <button onclick="scrollTopFunction()" id="scrollToTopButton" title="Go to top"><img src="images/arrows.png" class="arrows"></button>
+    <!-- ######################################### HEADER ######################################### -->
+    <div class="header" id="header">
+      <div id="header_logo_dark">
+        <img src="images/logo-2.svg" id="header_logo_dark">
+      </div>
+      <div id="header_logo">
+        <img src="images/logo.svg" id="header_logo">
+      </div>
+      <div class="header_item digital">
+        Digital Agency
+        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
+      </div>
```

Comment:
Nie jestem zwolennikiem tego żeby zostawiać luźny tekst pomiędzy znacznikami, ale też nie sprawdzałem jakie są dobre praktyki z tym związane. Poza tym zazwyczaj w menu się coś klika, a tutaj nie ma linka żadnego ani buttona.

```suggestion
      <div class="header_item digital">
        <a href="/">Digital Agency</a>
        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
      </div>
```

lub

```suggestion
      <a href="/" class="header_item digital">
        <span>Digital Agency</span>
        <img src="images/down-arrow.png" alt="arrow" class="header_item_arrow">
      </a>
```

Ale zgodnie z best practices raczej to drugie. To samo w innych miejscach.
Ogólnie to powinno być na znaczniku `nav` + `ul`.

Przykład:
```html
<nav>
  <ul>
    <li><a href="">Link 1</a></li>
    <li><a href="">Link 2</a></li>
    <li><a href="">Link 3</a></li>
  </ul>
  <button class="header_menu">
     ...
   </button>
</nav>
```